desktop/apps/editorial_features/routes.coffee

```diff
@@ -104,11 +115,19 @@ proxy = httpProxy.createProxyServer(changeOrigin: true, ignorePath: true)
   proxy.web req, res, target: target, (err) ->
     res.redirect 301, '/articles' if err
 
-setVideoIndex = (curation, slug) ->
+setVeniceVideoIndex = (curation, slug) ->
   for section, i in curation.get 'sections'
     if section.slug is slug
       return i
 
+setGucciVideoIndex = (slug) ->
+  switch slug
+    when 'present'
+      return 1
```

Comment:
FWIW Coffeescript's _everything is an expression_ allows implicit returns, so this could be:

```coffeescript
switch slug
  when 'present' then 1
  when 'future' then 2
  else 3
```

Reply:
Thanks! Will fix